### ISSUE TITLE
Links in error messages have poor contrast

### ISSUE THREAD

**Issue by fuzzbomb** [thememachine/eleven#159]
# The issue is:
Links inside of `.messages-error` have poor contrast.  It should have 4.5:1 contrast to meet [WCAG AA 1.4.1 Contrast (minimum)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

yellowsnow on strawberry-light has 2.8:1 contrast.
https://leaverou.github.io/contrast-ratio/#rgb%28252%2C%20206%2C%2084%29-on-rgb%28237%2C%2084%2C%20101%29

# Screenshots:
![eleventheme-contrast-messages-error](https://user-images.githubusercontent.com/1243005/28938430-f1b71a5a-7885-11e7-8753-bc771f786f35.png)

# Todo
Increase the contrast by making the background lighter, or the link text darker, or tweak both a bit.
